### PR TITLE
Add @jrbourbeau as a maintainer

### PR DESCRIPTION
**Pull request by jrbourbeau**
cc @mrocklin @jakirkham 

Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   sha256: 14f71c674c9d620c4d5a3483e39917c5ffb7012ecf6f31081d75efa8452a86f6
 
 build:
-  number: 0
+  number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml:12]: No need for the build number bump.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml (hunk 1 above)
Comment (jrbourbeau) [reply recipe/meta.yaml:12]: Ah, I see. Should I change it back to `0` or is the `1` okay, just not necessary?

**Comment by jakirkham** [comment]
It can be left as-is. Just a note for the future 😉

**Comment by jakirkham** [comment]
Thanks @jrbourbeau! 😀

**Comment by jrbourbeau** [comment]
Thanks for the pro-tip!

**Pull request merged**